docs/farming-&-staking/staking/operators/tips-operator.mdx

```diff
@@ -61,6 +47,10 @@ Only the account who registered **Operator** can swith the domain.
 
 :::
 
+:::note
+You need to update the move your keypain to a new location, if you switch domains. Follow the [insert key to keystore section](#insert-key-to-keystore)
```

Comment:
```suggestion
You need to move your keypair to a new location if you switch domains. Follow the [insert key to keystore section](#insert-key-to-keystore) section.
```

Reply:
keypain, probably how I was feeling at the moment 😆

Reply:
Didn't include the word "section" as it's already included in the link.